dist/angular-patternfly.js

```diff
@@ -1978,6 +1990,176 @@ angular.module('patternfly.select', []).directive('pfSelect', function ($timeout
     }
   };
 });
+;
+/**
+ * @ngdoc directive
+ * @name patternfly.utils.directive:pfTransclude
+ * @restrict A
+ * @element ANY
+ * @param {string} pfTransclude specifies the type of transclusion to use.<br/>
+ * <strong>Values:</strong>
+ * <ul style='list-style-type: none'>
+ * <li> 'sibling' - The transcluded contents scope is a sibling one to the element where transclusion happens (default)
+ * <li> 'parent'  - The transcluded contents scope is that of the element where transclusion happens.
+ * <li> 'child'   - The transcluded contents scope is child scope to the scope of the element where transclusion happens.
+ * </ul>
+ *
+ * @description
+ *   Directive for transcluding in directives and setting up scope of children of parent directives. This is a workaround
+ *   for https://github.com/angular/angular.js/issues/5489
+ *
```

Comment:
I see this as a utility that will be used time and again as add more complex directives.  This makes it easy on the directives and it's better to have it all in one place in my opinion.